### ISSUE TITLE
Fatal error: require(): Failed opening required 'Phpfastcache' folder

### ISSUE THREAD

**Issue by nikmauro** [khoaofgod/phpfastcache#189]
Warning: require(/home/www/mysite/vendor/phpfastcache/phpfastcache/src/Phpfastcache/legacy/aliases.php): failed to open stream: No such file or directory in /home/www/mysite/vendor/composer/autoload_real.php on line 55

Fatal error: require(): Failed opening required '/home/www/mysite/vendor/phpfastcache/phpfastcache/src/Phpfastcache/legacy/aliases.php' (include_path='.:/usr/share/pear:/usr/share/php') in /home/www/mysite/vendor/composer/autoload_real.php on line 55

This wrong path  /home/www/mysite/vendor/phpfastcache/phpfastcache/src/Phpfastcache/legacy/aliases.php 
the folder Phpfastcache is phpfastcache

**Comment by nikmauro** [comment]
I have rename manual the folder path and now the project return this 

Fatal error: Class 'Phpfastcache\core\InstanceManager' not found in /home/www/cms39/library/cache.php on line 53

I test with code in examples/3.setup.php

**Comment by nikmauro** [comment]
After composer update the the file /vendor/composer/autoload_prs4.php is empty

```php
<?php

// autoload_psr4.php @generated by Composer

$vendorDir = dirname(dirname(__FILE__));
$baseDir = dirname($vendorDir);

return array();

```

**Comment by Geolim4** [comment]
​Did you ran the composer ?​

2016-02-05 11:02 GMT+01:00 nikmauro <notifications@github.com>:

> anual the folder path and now the project return this




Georges.L

**Comment by nikmauro** [comment]
composer.json

```JSON
{
  "require": {
    "phpfastcache/phpfastcache": "dev-final"
  }
}
```

`# composer update
Loading composer repositories with package information
Updating dependencies (including require-dev)
  - Installing phpfastcache/phpfastcache (dev-final 3cca8f8)
    Cloning 3cca8f84903aff1faec67dd39a63fef852b992fc

Writing lock file
Generating autoload files`


autoload_files.php
```php
$vendorDir = dirname(dirname(__FILE__));
$baseDir = dirname($vendorDir);

return array(
    'b4d7f4479c3aadd86b8a9e7699412297' => $vendorDir . '/phpfastcache/phpfastcache/src/Phpfastcache/legacy/aliases.php',
);

```

autoload_namespaces.php

```PHP
$vendorDir = dirname(dirname(__FILE__));
$baseDir = dirname($vendorDir);

return array(
    'Phpfastcache' => array($vendorDir . '/phpfastcache/phpfastcache/src'),
);

```

**Comment by Geolim4** [comment]
Could you zip me the whole directory including phpfastcache and the composer bin ?
Thanks !

**Comment by Geolim4** [comment]
I also retested it in my side with a composer update, everything works fine, i'm surprised O.o

**Comment by nikmauro** [comment]
@Geolim4  https://www.wetransfer.com/downloads/c3b90b7bee0954e955ee9a91eae57d1520160205191615/10fe6a496881290b175bcdd154d1d08020160205191615/6295f1

**Comment by nikmauro** [comment]
@saiful [](url)

**Comment by Geolim4** [comment]
Ok got it, i'm on it.

**Comment by Geolim4** [comment]
Fixed with 4.0.3.
Delete your vendor directory and run a composer update :)

Regards,
Georges

**Comment by nikmauro** [comment]
Perfect! Thanks @Geolim4
